misk/web/tabs/web-actions/src/containers/WebActionsContainer.tsx

```diff
@@ -289,80 +257,18 @@ export const FilterWebActions = (props: IState & IDispatchProps) => {
   )
 }
 
-export const WebActionsContainer = (props: IState & IDispatchProps) => {
-  const tag = "WebAction"
-  const rawMetadata = simpleSelect(
-    props.simpleNetwork,
-    tag,
-    "webActionMetadata",
-    simpleType.array
-  )
-  const metadata = chain(rawMetadata)
-    .map((action: IWebActionAPI) => {
-      const authFunctionAnnotations = action.functionAnnotations.filter(
-        a => a.includes("Access") || a.includes("authz")
-      )
-      const nonAccessOrTypeFunctionAnnotations = action.functionAnnotations.filter(
-        a =>
-          !(
-            a.includes("RequestContentType") ||
-            a.includes("ResponseContentType") ||
-            a.includes("Access") ||
-            a.includes("authz") ||
-            a.toUpperCase().includes(HTTPMethod.DELETE) ||
-            a.toUpperCase().includes(HTTPMethod.GET) ||
-            a.toUpperCase().includes(HTTPMethod.HEAD) ||
-            a.toUpperCase().includes(HTTPMethod.PATCH) ||
-            a.toUpperCase().includes(HTTPMethod.POST) ||
-            a.toUpperCase().includes(HTTPMethod.PUT)
-          )
-      )
-      const emptyAllowedArrayValue =
-        authFunctionAnnotations.length > 1 &&
-        authFunctionAnnotations[0].includes("Unauthenticated")
-          ? "All"
-          : "None"
-      const allowedRoles =
-        action.allowedRoles.length > 0
-          ? action.allowedRoles.join(", ")
-          : emptyAllowedArrayValue
-
-      const allowedServices =
-        action.allowedServices.length > 0
-          ? action.allowedServices.join(", ")
-          : emptyAllowedArrayValue
-
-      return {
-        ...action,
-        allowedRoles,
-        allowedServices,
-        authFunctionAnnotations,
-        dispatchMechanism: [action.dispatchMechanism],
-        function: action.function.split("fun ").pop(),
-        nonAccessOrTypeFunctionAnnotations
-      }
-    })
-    .groupBy("pathPattern")
-    .map((actions: IWebActionInternal[]) => {
-      const dispatchMechanism = flatMap(
-        actions,
-        action => action.dispatchMechanism
-      )
-      const mergedAction = actions[0]
-      mergedAction.dispatchMechanism = dispatchMechanism.sort().reverse()
-      return mergedAction
-    })
-    .sortBy(["name", "pathPattern"])
-    .value()
-  // TODO(adrw) build index of keyspace for filterable fields
-  // const index = chain(metadata).value()
-  // console.log(index)
-  return (
-    <div>
-      <WebActions metadata={metadata} {...props} />
-      {/* <FilterWebActions {...props} /> */}
-    </div>
-  )
+const WebActionsContainer = (props: IState & IDispatchProps) => {
+  const metadata = simpleSelect(props.webActions, "metadata")
+  if (metadata.length > 0) {
+    return (
+      <div>
+        <WebActions metadata={metadata} {...props} />
+        {/* <FilterWebActions {...props} /> */}
```

Comment:
this is confusing, is this commented out?

Reply:
Yes commenting with `JSX` is a bit confusing.

Reply:
so this is a comment, why are we leaving this here?

Reply:
Will remove 👍